server/package.json

```diff
@@ -22,7 +23,8 @@
     "morgan": "^1.9.1",
     "multer": "^1.4.1",
     "request": "^2.88.0",
-    "uuid": "^3.3.2"
+    "uuid": "^3.3.2",
+    "web3": "1.0.0-beta.37"
```

Comment:
squid-js has web3 defined under `dependencies` so no need to add it here I think, just risks potential version mismatches

Reply:
Fixed in dd81201

Reply:
Build fails without web3

Reply:
FIxed with package-lock update